scripts/open-coredump.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+# Script for "one-click" opening of coredumps.
+# It extracts the build-id from the coredump, retrieves metadata for that
+# build, downloads the binary package, the source code and finally
+# launches the dbuild container, with everything ready to load the coredump.
+# The script is reentrant: running it after the prepartory steps will re-use what
+# is already donwloaded. It is *strongly* recommended to run this from an empty
+# directory, with nothing but the core-file present.
+#
+# Requirements:
+# * eu-unstrip
+# * curl
+# * jq
+# * git
+
+set -e
+
+SCYLLA_S3_RELOC_SERVER_URL="http://backtrace.scylladb.com"
+
+if [[ $# != 1 ]]; then
+	echo Please provide the path to the coredump
+	exit 1
+fi
+
+for required in eu-unstrip jq curl git; do
+	if ! type $required >& /dev/null; then
+		echo Please install $required first
+		exit 1
+	fi
+done
+
+COREFILE=$1
+
+BUILD_ID=$(eu-unstrip -n --core ${COREFILE} | cut -f2 -d' ' | cut -f1 -d@)
+
+echo "Build id: ${BUILD_ID}"
+
+BUILD=$(curl -s -X GET "${SCYLLA_S3_RELOC_SERVER_URL}/build.json?build_id=${BUILD_ID}")
```

Comment:
Maybe provide a way to pass this as an option of through the environment in case SCYLLA_S3_RELOC_SERVER is down or not working properly

Reply:
actually we should care about the s3 url for the reloc build package not this json object

Reply:
`PACKAGE_URL`, right?

Reply:
The thing is, if `backtrace.scylladb.com` is down, I have no way to get this information and pass it down. So it won't help.

Reply:
> actually we should care about the s3 url for the reloc build package not this json object

We use way more than just that, see below.